src/decode.rs

```diff
@@ -0,0 +1,41 @@
+use crate::{Error, JubJubAffine, JubJubScalar};
+use subtle::CtOption;
+
+// TODO: move this logic (in this form or another) to jubjub repo, or higher
+// repo than `dusk-pki`
+
+pub trait From32Bytes {
+    fn from_bytes(bytes: [u8; 32]) -> CtOption<Self>
```

Comment:
`CtOption` is very unstable and lack most of the benefits from the standard `Option`. If we are to use that as standard only because of conditional selection, then maybe we should think on a solution to encapsulate the conditional selection instead of setting subtle as our standard.

Reply:
I'm not using `CtOption` because I want to, it's because the `from_bytes` impl of JubJub's return it, so the trait needs to be consistent with that.

AFAIK @CPerezz is working on a PR for the subtle crate that will implement a conversion from `CtOption` to `Option`. So basically this is just a temporary measure.

Reply:
Yes, I'll try to do it today at night or so. Henry already agreed on the idea I had. So it's a matter of merge the PR and then wait for a release to have this conversion finally avaliable.